### ISSUE TITLE
Use another temporary directory for HTML pages

### ISSUE THREAD

**Issue by maaaks** [chris-griffin/flask-wkhtmltopdf#2]
Currently, Flask-WkHTMLtoPDF uses `os.path.dirname(__file__)` as the temporary directory for generating HTML pages. It causes access error:

>PermissionError: [Errno 13] Permission denied: '/usr/local/lib/python3.5/dist-packages/tmp0kurpn6l.html'

I believe you should use a [`TemporaryDirectory`](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) or make this path configurable, much like `PDF_DIR_PATH`.

**Comment by maaaks** [comment]
Or, maybe just use `PDF_DIR_PATH` itself for this, too.

**Comment by chris-griffin** [comment]
Yes, that makes sense to pull out into a setting. Can you submit a PR for the change?
